Resources/Prototypes/_White/Entities/Objects/Weapons/Guns/Turrets/ship_turrets.yml

```
@@ -0,0 +1,171 @@
+  - type: Clickable
+  - type: InteractionOutline
+  - type: Actions
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.45,-0.45,0.45,0.45"
+        density: 60
+        mask:
+          - MachineMask
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)